source/Modules/Controller/connection.js

```diff
@@ -1,14 +1,14 @@
 import qs from 'query-string';
 import extension from 'extensionizer';
 import ERRORS from '@background/errors';
-import PlugController from '@psychedelic/plug-controller';
 import { validatePrincipalId } from '@shared/utils/ids';
 import { CONNECTION_STATUS } from '@shared/constants/connectionStatus';
+import { fetchCanistersInfo, initializeProtectedIds } from '@background/utils';
```

Comment:
Will add this method to the `BackgroundModule` class so we don't have to import them +1